Automattic/sensei
# Update npm packages

Update all npm packages to the latest versions that are compatible within `package.json`. Note that none of our node packages are used outside of the build process at this point.

Also see p6rkRX-HD-p2

## Testing instructions

- Run the build with `gulp` and ensure all JS minification and SCSS compilation behaves as expected.
